Combobox.js

```diff
@@ -960,8 +1000,10 @@ define([
 		_dropDownKeyUpHandler: dcl.superCall(function (sup) {
 			return function () {
 				if (this.hasDownArrow) {
-					if (this.inputNode.value.length === 0 && this.minFilterChars === 0) {
-						this.list.query = this._getDefaultQuery();
+					if (this.inputNode.value.length === 0) {
+						if (this.minFilterChars === 0) {
+							this.list.query = this._getDefaultQuery();
+						}
```

Comment:
- What if the list hasn't been created yet? Didn't we agree to always set the list's query at the same time as we create the list? 
-- Sure, I should check if the list is already created. However `this.list.query` and `this.list.source` are set at the same time, because the `_dropDownKeyUpHandler` function calls its `sup` function which calls `openDropDown` (where this.list.source gets assigned. So, what's wrong?

- What if the user typed a different key than the down arrow, such as backspace?
-- what's your point? In this case the above method (`_dropDownKeyUpHandler`) runs only when the user press the Down Arrow Key (see: https://github.com/ibm-js/delite/blob/master/HasDropDown.js#L400-L412). So, what's the problem with backspace?